src/js/actions/documents.js

```diff
@@ -273,15 +273,12 @@ define(function (require, exports) {
      * Initialize document and layer state, emitting DOCUMENT_UPDATED events, for
      * all the inactive documents.
      *
-     * @param {number} currentIndex
+     * @param {number} currentDocID
      * @param {number} docCount
      * @return {Promise}
      */
-    var initInactiveDocuments = function (currentIndex, docCount) {
+    var initInactiveDocuments = function (currentDocID, docCount) {
         var otherDocPromises = _.range(1, docCount + 1)
-            .filter(function (index) {
```

Comment:
You don't want to call `_getInactiveDocumentByRef` on the active document. This is the reason why we used the index in the first place. For this reason (i.e., to avoid an extra round-trip call to PS) I'm not sure we want to take this change.